fluster/main.py

```diff
@@ -47,23 +49,57 @@ class Main:
     def __init__(self) -> None:
         self.decoders_dir = DECODERS_DIR
         self.test_suites_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), TEST_SUITES_DIR
-        )
-        # Only use the system directory for test suites if the local directory
-        # doesn't exist and the system directory does exist.
-        if (
-            sys.platform.startswith("linux")
-            and not os.path.exists(self.test_suites_dir)
-            and os.path.exists(TEST_SUITES_DIR_SYS)
-        ):
-            self.test_suites_dir = TEST_SUITES_DIR_SYS
-
+            os.path.dirname(__file__), "..", TEST_SUITES_DIR
+        )
         self.resources_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESOURCES_DIR
+            os.path.dirname(__file__), "..", RESOURCES_DIR
         )
-        self.results_dir = os.path.join(
-            os.path.join(os.path.dirname(__file__), ".."), RESULTS_DIR
+        self.results_dir = os.path.join(gettempdir(), RESULTS_DIR)
+
+        is_uninstalled = os.path.exists(
+            os.path.join(os.path.dirname(__file__), "..", ".git")
         )
+        if not is_uninstalled:
+            appname = "fluster"
+            site_data_dir = platformdirs.site_data_dir(appname)
+            user_data_dir = platformdirs.user_data_dir(appname)
+
+            test_suites_dir = os.path.join(site_data_dir, TEST_SUITES_DIR)
+            if os.path.exists(test_suites_dir):
+                self.test_suites_dir = test_suites_dir
+            else:
+                self.test_suites_dir = os.path.join(user_data_dir, TEST_SUITES_DIR)
+
+            resources_dir = os.path.join(site_data_dir, RESOURCES_DIR)
```

Comment:
site_data_dir as it's implemented will not work when fluster is installed as a PyPI package.

Reply:
Fixed in f409dcb using always user_data_dir as described in prev thread.